### PR TITLE
Add `Reduced Motion` to docs

### PR DESCRIPTION
**Pull request by bartlomiejbloniarz**
## Summary

This PR extends the docs with information about the api related to `Reduced Motion`. I changed the `docs/package.json` to use a specific nightly build, so the docs can be released immediately.

### CODE REVIEW

File: docs/src/components/InteractivePlayground/useSpringPlayground/index.tsx [hunk 1]
```diff
@@ -158,6 +161,12 @@ export default function useSpringPlayground() {
         value={restSpeedThreshold}
         onChange={setRestSpeedThreshold}
       />
+      <SelectOption
+        label="Reduce Motion"
```
Comment (kacperkapusciak) [inline docs/src/components/InteractivePlayground/useSpringPlayground/index.tsx:165]:
```suggestion
        label="Reduce motion"
```

File: docs/src/components/InteractivePlayground/useTimingPlayground/index.tsx [hunk 2]
```diff
@@ -312,6 +315,12 @@ export default function useTimingPlayground() {
           />
         </>
       )}
+      <SelectOption
+        label="Reduce Motion"
```
Comment (kacperkapusciak) [inline docs/src/components/InteractivePlayground/useTimingPlayground/index.tsx:319]:
```suggestion
        label="Reduce motion"
```

File: docs/src/components/InteractivePlayground/useSpringPlayground/Example.tsx [hunk 3]
```diff
@@ -16,6 +18,18 @@ interface Props {
 
 export default function App({ options }: Props) {
   const offset = useSharedValue(initialOffset);
+  const shouldReduceMotion =
+    options.reduceMotion === ReduceMotion.Always ||
+    (useReducedMotion() && options.reduceMotion === ReduceMotion.System);
```
[the remark this answers is not in the document]
Comment (bartlomiejbloniarz) [reply docs/src/components/InteractivePlayground/useSpringPlayground/Example.tsx:23]: I can reorder it a bit so it's easier to read, but I don't see how I could further simplify
```js
const reduceMotion = useReducedMotion();
const shouldReduceMotion =
    options.reduceMotion === ReduceMotion.Always ||
    (options.reduceMotion === ReduceMotion.System && reduceMotion);

**Review by kacperkapusciak — APPROVED** [review]
:shipit:

**Pull request merged**